Bottom: 1801eac9cf7ae938e37f2283fd9e428216376a76
Top:    61af7b78e1666f92ff6c99f3fc62a74188655b12
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 15:36:48 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 57da8e6..e4ea97d 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -279,6 +279,9 @@ static int rpc_do_create_client(struct sockaddr *addr, struct conn_info *info, i
 		laddr = (struct sockaddr *) &in4_laddr;
 		in4_raddr->sin_port = htons(info->port);
 		slen = sizeof(struct sockaddr_in);
+		/* Use rpcbind v2 for AF_INET */
+		if (info->program == rpcb_prog)
+			info->version = PMAPVERS;
 	} else if (addr->sa_family == AF_INET6) {
 		struct sockaddr_in6 *in6_raddr = (struct sockaddr_in6 *) addr;
 		in6_laddr.sin6_family = AF_INET6;
